tests/src/e2e.spec.ts

```diff
@@ -35,7 +35,7 @@ afterAll(async () => {
   await electronApp.close();
 });
 
-test.skip('Check welcome page is displayed and that we are redirected to the main page where dashboard page is there', async () => {
```

Comment:
There was a reason that the test was skipped, right?

Reply:
There was a reason, I though it was instability. This is just to explain the inital proposal.

Reply:
yes instability for 1.0 release

Reply:
ok.